ci/rpc_client.py

```
@@ -191,22 +208,69 @@ async def connect(self, boot_wait: float = 3.0, drain_boot: bool = True) -> None
+                        f"skipping fixed sleep."
+                    )
+                else:
+                    print(
+                        f"⏳ [RPC] No boot signal in {boot_wait}s "
+                        f"(drained {len(poll_drained_lines)} line(s)); "
+                        f"falling back to timeout behavior."
+                    )
 
         if drain_boot:
             if self.verbose:
                 print(f"📥 [RPC] Draining boot output...")
```

Ruff (0.15.10)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)